web-src/src/components/ModalDialogAlbum.vue

```diff
@@ -12,6 +12,10 @@
               <p class="title is-4">
                 <a class="has-text-link" @click="open_album">{{ album.name }}</a>
               </p>
+              <div class="buttons" v-if="media_kind === 'podcast'">
+                <a class="button is-small" v-if="new_tracks > 0" @click="mark_played">Mark as played</a>
```

Comment:
On the `podcasts` page, when this dialog is brought up it does not allow us to _Mark as played_ as it doesn't have a `new_tracks` reference
```
v-if="new_tracks === undefined || new_tracks > 0"
```
Can we have this _Mark as Played_ displayed unconditionally here?  If there's no unplayed tracks theres no updates to the db.  In the indivudla `podcast` page, this button would only exit if its really got new items.

The use case for this:

if I add a number of new RSS podcasts at the same time/(adjust the limit of an a few existing subscriptions)[https://github.com/ejurgensen/forked-daapd/issues/951#issuecomment-615392753], the `podcasts` page may have a long list of unplayed items.  I want to mark played for individual podcasts without going into the `podcast` page to do this.

Reply:
Yes, makes sense. I'll change that.